wme-address-point-helper.user.js

```diff
@@ -68,7 +69,8 @@
         addNavigationPoint: 'Додавати точку в\'їзду',
         inheritNavigationPoint: 'Наслідувати точку в\'їзду від POI',
         autoSetHNToName: 'Копіювати номер будинку в назву',
-        noDuplicates: 'Не створювати дублікатів'
+        noDuplicates: 'Не створювати дублікатів',
+        CopyPOI: 'Копіювати POI як точку',
```

Comment:
```suggestion
        copyPOI: 'Копіювати POI як точку',
```
fix all over the code, please)